packages/flutter_localizations/lib/src/l10n/material_no.arb

```diff
@@ -48,9 +49,7 @@
   "reorderItemRight": "Flytt til høyre",
   "expandedIconTapHint": "Skjul",
   "collapsedIconTapHint": "Vis",
-  "remainingTextFieldCharacterCountZero": "TBD",
```

Comment:
If this file was just renamed, why did the messages it contains change?

Reply:
My description was misleading, I actually ran the translation console script to regenerate the `_no.arb` files in case anything had changed. I'm not sure why this would be removed instead of saying 'TBD'. Do we want to keep it as-is, or re-introduce the original arb file setup?

Reply:
I see, didn't realize we were updating translations. Removing this line is OK; just means that there's no special case for zero.

Reply:
Turns out there was a manual step to introduce the 'TBD' for any message that does not have a translation in google's translation console. I re-introduced the removed 'TBD' string.